Fix HGNetV2 Model Card and Image Classification Pipeline Usage Tips (#39965)
huggingface/transformers: docs/source/en/model_doc/hgnet_v2.md

-->

<div style="float: right;">
    <div class="flex flex-wrap space-x-1">
        <img alt="PyTorch" src="https://img.shields.io/badge/PyTorch-DE3412?style=flat&logo=pytorch&logoColor=white">
    </div>
</div>

# HGNet-V2

> [HGNetV2 (High Performance GPU Network V2)](https://github.com/PaddlePaddle/PaddleClas/blob/v2.6.0/docs/zh_CN/models/ImageNet1k/PP-HGNetV2.md) is a next-generation convolutional neural network (CNN) backbone developed by the **PaddlePaddle Vision Team at Baidu**, purpose-built for optimal accuracy-latency tradeoff on NVIDIA GPUs. Building on the original [HGNet](https://github.com/PaddlePaddle/PaddleClas/blob/v2.6.0/docs/zh_CN/models/ImageNet1k/PP-HGNet.md), HGNetV2 delivers high accuracy at fast inference speeds and performs strongly on tasks like image classification, object detection, and segmentation, making it a practical choice for GPU-based computer vision applications.

Reviewer: ```suggestion
[HGNetV2](https://github.com/PaddlePaddle/PaddleClas/blob/v2.6.0/docs/zh_CN/models/ImageNet1k/PP-HGNetV2.md) is a next-generation convolutional neural network (CNN) backbone built for optimal accuracy-latency tradeoff on NVIDIA GPUs. Building on the original [HGNet](https://huggingface.co/papers/2407.01097), HGNetV2 delivers high accuracy at fast inference speeds and performs strongly on tasks like image classification, object detection, and segmentation, making it a practical choice for GPU-based computer vision applications.
```

> This model was contributed by [VladOS95-cyber](https://github.com/VladOS95-cyber). 
> The original code can be found [here](https://github.com/PaddlePaddle/PaddleDetection/blob/develop/ppdet/modeling/backbones/hgnet_v2.py).

Reviewer: ```suggestion
You can find all the original HGNet V2 models under the [USTC](https://huggingface.co/ustc-community/models?search=hgnet) organization.
```

> > [!TIP]

Reviewer: ```suggestion
> [!TIP]
> This model was contributed by [VladOS95-cyber](https://github.com/VladOS95-cyber).
```
